test/Common/CustomTestStorageAccountProvider.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using DurableTask.AzureStorage;
+using Microsoft.WindowsAzure.Storage;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Tests
+{
+    internal class CustomTestStorageAccountProvider : IStorageAccountProvider
+    {
+        private const string CustomConnectionString = "DefaultEndpointsProtocol=https;AccountName=test;AccountKey=dGVzdA==;EndpointSuffix=core.windows.net";
```

Comment:
Is it possible to run these tests with an account key that is randomly generated rather than hardcoding one that looks semi-real? I'm concerned that our repo may get flagged by code scanners looking for credential leaks.

Reply:
Done. Changed to use random key for the storage account key.